packages/editor/src/client/locales/en.json

```diff
@@ -3,6 +3,7 @@
     "articleEditor": {
       "overview": {
         "articlePublished": "Article Published",
+        "articlePending": "Article pending",
```

Comment:
Really small comment, maybe change it to Pending (capital) just so it's consistent with the Article Published format.

Reply:
Done